### ISSUE TITLE
Add a Redirect Block 

### ISSUE THREAD

**Issue by erral** [plone/volto#3303]
**Describe the bug**

If we build a content structure like this:

/Plone/communications-department
/Plone/communications-department/news
/Plone/communications-department/events
/Plone/communications-department/press-releases

In Plone we can add a "Link" item, set it as a "default page" for `communications-department` folder and get a redirect to, say, `/Plone/communications-department/news`, this way we can group several folders under a folder and get the users redirected.

In plain Volto this is not possible.

A "Redirect Block" could help here. If we add a "Redirect Block" to the `communications-department` then it will work like in Classic Plone.

**Comment by pbauer** [comment]
Why not use a folderish Document instead of a Folder? Just to keep the url with `/news`?

**Comment by erral** [comment]
Yes indeed we can, I was trying to bring to Volto what we currently have in Plone with "links-as-default-pages" which can't be done right now in Volto.
